### PR TITLE
Add Subjugation - Slavery and Capture Mechanics

### PR DESCRIPTION
**Pull request by Superfly-Johnson**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1718,6 +1718,27 @@ plugins:
       - <<: *obsolete
         subs: [ '[Better Game Performance v5](https://www.nexusmods.com/newvegas/mods/51871/)' ]
 
+  - name: 'SD_Capture( TTW Additions)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/82597'
```
Comment (pStyl3) [inline masterlist.yaml]: Please add a trailing slash `/` at the end of the link.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

**Comment by pStyl3** [comment]
Also going forward, please state which version of a plugin was used while cleaning it.

It would also be good, if you use the latest non-`extremely experimental` version of xEdit for adding cleaning data. Right now that would be xEdit 4.0.4c - see [this Discord link](https://discord.com/channels/471930020454072348/518048160526893057/1011479030157029426), the `4.0.4c` build can be found in the `xedit-builds` channel of their Discord server.

File: masterlist.yaml (hunk 1 above)
Comment (pStyl3) [reply masterlist.yaml]: Your latest commit added a `é` instead of a trailing slash `/` at the end of the url.

Comment (Superfly-Johnson) [reply masterlist.yaml]: Canadian french keyboard strikes again. I'll fix it later tonight, not to worry

**Review by pStyl3 — APPROVED** [review]

**Comment by pStyl3** [comment]
Thanks

**Pull request merged**